fn_ipvoid/fn_ipvoid/lib/resilient_common.py

```diff
@@ -0,0 +1,26 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# pragma pylint: disable=unused-argument, no-self-use
+BASE_URL = 'https://endpoint.apivoid.com/{}/v1/pay-as-you-go/?key={}&{}'
```

Comment:
```
https://endpoint.apivoid.com/
```
Should be an app.config variable. You will need to add it to the `config.py` file

See: https://github.com/neetinkandhare/resilient-community-apps/blob/alien-vault/fn_alienvault_otx/fn_alienvault_otx/util/config.py#L17